demo/views.py

```diff
@@ -2,7 +2,14 @@
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
 
 from wagtail.wagtailcore.models import Page
-from wagtail.wagtailsearch.models import Query, EditorsPick
+from wagtail.wagtailsearch.models import Query
+
+try:
+    # Wagtail >= 1.1
+    from wagtail.contrib.wagtailsearchpromotions.models import SearchPromotion
+except ImportError:
+    # Wagtail < 1.1
+    from wagtail.wagtailsearch.models import EditorsPick as SearchPromotion
```

Comment:
It means that we can merge this change now, before any 1.1 releases exist (and thus make wagtaildemo usable for development of 1.1 again - currently it isn't).